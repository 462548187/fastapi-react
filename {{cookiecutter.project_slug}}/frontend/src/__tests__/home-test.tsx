it('Example test', () => {
  expect(true);
});
