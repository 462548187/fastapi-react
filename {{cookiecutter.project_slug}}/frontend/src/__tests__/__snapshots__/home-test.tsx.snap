// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
Array [
  <a
    className="makeStyles-link-1"
    onClick={[Function]}
  >
    Click to make request to backend
  </a>,
  "",
  "",
  <a
    className="makeStyles-link-1"
    href="/admin"
  >
    Admin Dashboard
  </a>,
  <a
    className="makeStyles-link-1"
    href="/protected"
  >
    Protected Route
  </a>,
  <a
    className="makeStyles-link-1"
    href="/login"
  >
    Login
  </a>,
]
`;
